server/src/test/antlr4/com/broadcom/usecase/UseCasePreprocessor.g4

```diff
@@ -130,7 +130,7 @@ MULTITOKENSTOP : '_' + STOP;
 STRINGLITERAL : ["] ~["]* ["];
 IDENTIFIER : [a-zA-Z0-9:]+ ([-_]+ [a-zA-Z0-9:]+)*;
 
-COPYBOOKNAME : [a-zA-Z0-9#@$]+ ([-_]+ [a-zA-Z0-9#@$]+)*;
+COPYBOOKNAME : ('\'')*([a-zA-Z0-9#@$]+ ([-_]+ [a-zA-Z0-9#@$]+)*)+('\'')*;
```

Comment:
```suggestion
COPYBOOKNAME : [a-zA-Z0-9#@$]+ ([-_]+ [a-zA-Z0-9#@$]+)*;
QUOTED_COPYBOOKNAME : '\'' + COPYBOOKNAME + '\'';
DOUBLE_QUOTED_COPYBOOKNAME : '"' + COPYBOOKNAME + '"';
```

I would recommend specifying explicitly the possible cases not to allow unclosed literals or something. It will also need to add this:
```
cpyName
   : IDENTIFIER | COPYBOOKNAME | QUOTED_COPYBOOKNAME | DOUBLE_QUOTED_COPYBOOKNAME
   ;
```

Reply:
Thanks! I just did not find a way to do that :-)